go.mod

```diff
@@ -0,0 +1,35 @@
+module github.com/openshift-online/async-routine
+
+go 1.23.6
```

Comment:
This is meant to be the minimum version required, but 1.23.6 is the latest version in F41, which means there will be a bunch of software behind this. Cluster Service itself is on 1.22.5 and I don't know about the pipelines we currently use.

Reply:
fair point, I'll adust.